src/app/actions/actions.tsx

```diff
@@ -0,0 +1,38 @@
+'use server'
+ 
+import { prisma } from '../../server/db';
+import { revalidatePath } from 'next/cache';
+import { schema } from '../components/ui/projectForm/ProjectForm';
+import { type z } from 'zod';
+
+type Project = z.infer<typeof schema>;
+
+export const addProject = async (data: Project) => {
+    await prisma.project.create({
+        data: {
+            name: data.name,
+            upworkId: data.upworkId,
+            hubstaffId: data.hubstaffId,
+            asanaId: data.asanaId,
+        },
```

Comment:
Why not just `create({ data })`? You don't need to write all the properties 1-by-1